GenomeFileUtil.spec

```diff
@@ -65,6 +65,38 @@ module GenomeFileUtil {
     funcdef genbank_to_genome(GenbankToGenomeParams params)
                 returns (GenomeSaveResult result) authentication required;
 
+    typedef structure {
+        File file;
+        string genome_name;
+
+        string source;
+        string taxon_wsname;
+        string taxon_id;
+
+        string release;
+        string generate_ids_if_needed;
+        int    genetic_code;
+        string scientific_name;
+        usermeta metadata;
```

Comment:
This looks like it already allows passing workspace object metadata, is that right? Or does that wind up somewhere else?

Reply:
> This looks like it already allows passing workspace object metadata, is that right? 

Yes

Reply:
Great, less work :+1: